packages/dropdown-react/src/Select.tsx

```diff
@@ -20,23 +23,43 @@ export function Select({
     inline = false,
     helpLabel,
     errorLabel,
+    variant,
+    placeholder,
+    value,
     ...rest
 }: Props) {
-    const handleChange = (event: React.ChangeEvent<HTMLSelectElement>) => onChange && onChange(event);
+    // Set value to value given, or to first item if no value or placeholder is given:
+    value = value ? value : placeholder ? "" : items[0];
```

Comment:
Nested turnary... I do not like... This should not pass linting. The cognative load of reading that line makes me sleepy.

Reply:
I'll refactor it 😅

Reply:
I think we should add a lint rule to avoid this in the future